json-to-go.js

```diff
@@ -16,7 +16,7 @@ function jsonToGo(json, typename)
 
 	try
 	{
-		data = JSON.parse(json);
+		data = JSON.parse(json.replace(/\.0/g, ".1"));
```

Comment:
Yep, I noted that in the PR. It *could* mess them up. But right now it messes up *every* float that has `.0`. So the risk seems to me outweighed by the benefit.